config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-x-on-y.yaml

```diff
@@ -0,0 +1,127 @@
+# periodic jobs
+
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kinder-master-on-stable
+  interval: 1h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decoration_config:
+    timeout: 2400000000000 #40m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: release-1.14
```

Comment:
so this means we are building both e2e and kubeadm-e2e from release-1.14, given we are testing the kubeadm binary from master, shouldn't we be testing this binary against kubeadm-e2e from master?

Reply:
Kubeadm master should be able to bootstrap a Kubernetes cluster v1.14
Then we are going to use E2E test suite v1.14 and E2E kubeadm test suite v1.14 to check that the Kubernetes cluster (v1.14 too) works properly

In other words, test should be of the same version of cluster I am testing, not of version the bootstrap tool